bpfman/src/bin/cli/args.rs

```diff
@@ -36,6 +36,10 @@ pub(crate) enum Commands {
     /// eBPF Bytecode Image related commands.
     #[command(subcommand)]
     Image(ImageSubCommand),
+    #[clap(hide = true)]
```

Comment:
On the fence, wondering if the commands should be hidden or not? We could expose later if we want, so not a blocker.